packages/now-static-build/build.sh

```diff
@@ -6,3 +6,6 @@ rm -rf dist
 
 # Build with `ncc`
 ncc build src/index.ts -e @vercel/build-utils -e @now/build-utils -o dist
+
+bridge_defs="$(dirname $(pwd))/now-node/dist/bridge.js"
+cp -v "$bridge_defs" dist
```

Comment:
Let's use the same pattern as the other runtimes:

```sh
bridge_defs="$(dirname $(pwd))/now-node-bridge/src/bridge.ts"

cp -v "$bridge_defs" src
```

Reply:
@styfle that means that we won't be able to use ncc for static-build, since we'd need to transform it to typescript, but not bundle it, since we just want to read it, would that be okay?

Reply:
@AndyBitz You need to add the missing `tsc` command like we do for node and next runtimes and it should work.